gammapy/irf/io.py

```diff
@@ -19,6 +13,45 @@
 }
 
 
+# The key is the class tag. The hdu name is not strictly declared in the GADF
```

Comment:
Technically the EXTNAME is provided by the HDU index no?
If there is no specification it might be unsafe to use one.

Reply:
Ah, indeed. I forgot about this. So this question only applies to writing IRFs and to provide a default for reading. I guess there we can use any internal Gammapy convention we want. So I'll just keep what we have and remove the comment...